playbooks/acs.yml

```diff
@@ -25,12 +25,15 @@
     secret_ca_passphrase: "{{ hostvars.localhost.ca_signing_key_passphrase }}"
   ansible.builtin.import_playbook: pki.yml
 
-- name: Gather facts from all hosts
+- name: Setup facts
   hosts: all:!external
   gather_facts: false
+  roles:
+    - role: "../roles/common"
   tasks:
-    - name: Setup
-      ansible.builtin.setup:
+    - name: Set alfresco url from fqdn_alfresco if available or localhost
+      ansible.builtin.set_fact:
+        alfresco_url: "{% if use_ssl | bool %}https{% else %}http{% endif %}://{% if fqdn_alfresco %}{{ fqdn_alfresco }}{% else %}localhost{% endif %}"
```

Comment:
Just because `fqdn_alfresco` is not set doesn't mean install is done on localhost. `fqdn_alfresco` is a param that was introduced for ssl configuration only and is documented as such.
Though, I understand what we want to do is not "conventional" and proper implementation would require more time. let's take some time to discuss it a bit further

Reply:
I got inspired by https://github.com/Alfresco/alfresco-ansible-deployment/blob/f96645fe04025c7e2ee6891e7483b29929f3db4c/roles/repository/templates/alfresco-global.properties.j2#L57 - I could try to reintroduce `nginx_host` for consistency, I used localhost just as a shortcut knowing that would work locally and on cloud-infra where we have ssl setup in place.

To summarize a little, I need a fact for the main url where clients can access the application because it needs to be known for keycloak redirect uri and clients needs to use the same otherwise the oauth issuer will not match.

Reply:
For this use case I would normally use `known_urls | first` but on cloud-infra that would mean an https url. Would it work with the current proxy setup?